### PR TITLE
MAGEDOC-3687: Amazon Sales Channel GA docs

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) removed beta banner and updated availability for asc ga

## Affected DevDocs pages

- https://devdocs.magento.com/extensions/amazon-sales/
- https://devdocs.magento.com/extensions/amazon-sales/release-notes/
- https://devdocs.magento.com/release/

whatsnew
Published Amazon Sales Channel documentation for GA release.
- [Install](https://devdocs.magento.com/extensions/amazon-sales/)
- [Release Notes](https://devdocs.magento.com/extensions/amazon-sales/release-notes/)
- [Compatibility](https://devdocs.magento.com/release/)

### CODE REVIEW

File: release/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The following table describes the status of Magento software availability and wh
...
 | **Page Builder**                                | Available now                                                                                                                                        | Bundled with {{site.data.var.ee}} 2.3.x                                                                                      |
-| **Amazon Sales Channel**                        | Early access closed<br><br>Regional availability May 10, 2019 (US, Canada, Mexico)<br><br>EMEA and APAC availability expected in second half of 2019 | Magento Marketplace                                                                                                          |
+| **Amazon Sales Channel**                        | Available now (US, Canada, Mexico)<br><br>EMEA and APAC availability expected in second half of 2019 | Magento Marketplace                                                                                                          |
```
Comment (jeff-matthews) [inline release/index.md]:
```suggestion
| **Amazon Sales Channel 2.0.0**                        | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x (US, Canada, Mexico)<br><br>EMEA and APAC availability expected in second half of 2019 | Magento Marketplace                                                                                                          |
```

Comment (hickskenh) [reply release/index.md]: accepted suggestion

**Comment by jeff-matthews** [comment]
Wait for authorization before running tests and merging.

**Comment by hickskenh** [comment]
running tests

**Pull request merged**